persistence-api/src/main/java/io/stargate/db/query/BoundInsert.java

```diff
@@ -0,0 +1,18 @@
+package io.stargate.db.query;
+
+import java.util.List;
+
+public interface BoundInsert extends BoundDMLQuery {
+  /**
+   * Like updates, inserted cannot impact range of rows, but contrarily to updates, inserts inserts
+   * can only impact a single row. This method is just a shortcut to returns that primary key.
+   */
+  default PrimaryKey primaryKey() {
+    assert !rowsUpdated().isRanges();
```

Comment:
That's the good old debate on assertions versus actual exceptions :).

Here, it's meant as a genuine code assertion: I'm pretty certain this can never trigger with current code and it's almost more meant as code documentation than anything else. I don't care much that someone may disable assertions in production here.

_But_ i know some people are strongly against assertions in Java, so I'm happy to remove it, or change it to an actual test if this bother you.

Reply:
I don't have strong preference, however, the raw `assert` seems not very usable.